projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -86,20 +86,44 @@
       </div>
     </div>
     <!-- Edit Mode Actions -->
-    <div *ngIf="editMode" class="row cx-card-body-delete">
-      <div class="col-md-6">
-        <button
-          [cxAtMessage]="'assistiveMessage.actionCancelled' | cxTranslate"
-          class="btn btn-block btn-secondary"
-          (click)="cancelEdit()"
-        >
-          {{ 'common.cancel' | cxTranslate }}
-        </button>
+    <div *ngIf="editMode">
```

Comment:
I could be wrong here looking at the diff. But it looks like you have removed `class="row cx-card-body-delete"` from the `div` with `*ngIf="editMode"` and placed it on the nested `div`. This would be a breaking change. Could you please verify this?

Reply:
I've updated it.